tests/test_filesystem.py

```diff
@@ -659,12 +659,12 @@ def test_virtual_filesystem_link(vfs_path, link_path):
         ),
     ],
 )
-def test_virtual_filesystem_symlink(vfs_path, link_path):
+def test_virtual_filesystem_symlink(vfs_path: str, link_path: str) -> None:
```

Comment:
Aren't these changes technically part of the `VirtualFileesystem` symlink fix commit?

Reply:
You're right, moved it over.